yarn.lock

```diff
@@ -279,20 +279,20 @@
     minimatch "^3.0.4"
     strip-json-comments "^3.1.1"
 
-"@eth-optimism/contracts@^0.5.5":
-  version "0.5.8"
-  resolved "https://registry.yarnpkg.com/@eth-optimism/contracts/-/contracts-0.5.8.tgz#975313a219b5537bad7f4f98f8841c98592882d0"
-  integrity sha512-INvIMiE72Y6tjxm+BZyN9f3+LiMKJgmNpR0qEimrvpg49hK8gOXQTnI+0C7/qIIGvgaQgDyejG9AfkxZizjLVQ==
+"@eth-optimism/contracts@^0.5.11", "@eth-optimism/contracts@^0.5.5":
```

Comment:
@nicholaspai the reason you didn't see changes in the yarn.lock (I think) was that this package was _already in the dependency tree_ with a permissible version, most likely due to the fact that we import uma/core. You could probably drop these yarn.lock changes if you wanted and I think everything would work (but nothing wrong with regenerating things at this early stage).

Reply:
interesting, what do you think we should do--keep the new yarn.lock or drop? Also should we update the eth-optimism version if its available?

Reply:
I think keeping the updates is totally fine! I just wanted to figure out what happened to make sure something wasn't broken in our setup or something :)